lib/msal-electron-proof-of-concept/src/AppConfig/PublicClientApplication.ts

```diff
@@ -13,4 +15,35 @@ export class PublicClientApplication extends ClientApplicationBase {
     constructor(authOptions: AuthOptions) {
         super(authOptions);
     }
+
+    /**
+     * The acquireToken method uses the Authorization Code
+     * Grant to retrieve an access token from the AAD authorization server,
+     * which can be used to make authenticated calls to an resource server
+     * such as MS Graph.
+     */
+    public acquireToken(request: AuthenticationParameters): string {
+        // Validate and filter scopes
+        this.validateInputScopes(request.scopes);
+        const scope = request.scopes.join(' ').toLowerCase();
```

Comment:
Whats the point of this line?

Reply:
I thought msal-core did it as pre-processing before adding them to the request URL but it turns out that string was actually used as an object key that we won't be using in msal-electron, so I'm removing it.